projects/packages/publicize/src/auto-conversion-settings/class-rest-settings-controller.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * Class used to register REST API auto-conversion settings endpoints.
+ *
+ * Flagged to be removed after deprecation.
```

Comment:
Let's use the `@deprecated` annotation for this. We can add something to the backlog, to make sure we come back to tidy this up